app/forms/concept_form.rb

```diff
@@ -0,0 +1,13 @@
+class ConceptForm
```

Comment:
@hweng I think this is a pattern used by the Hydra Editor to draw the form. And it'll be simple to add another class with such a form definition to the frame work.